### PR TITLE
Add operations section

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Adds a new section to Timescale Cloud for the Operations dashboard. Includes a procedure for forking.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes https://github.com/timescale/docs/issues/475

### CODE REVIEW

**Comment by Loquacity** [comment]
> https://docs-dev.timescale.com/docs-db-forking-lana/cloud/db-forking-lana/operations/#forking-a-service
> 
> We should add the availability for forking.
> 
> Either Paused or Running services can be forked however services resuming or "in progress" cannot be forked until they complete their transition.

Sure. I need access to the feature before I can really round this out.

**Comment by Loquacity** [comment]
image

**Review by clarkbw — APPROVED** [review]
💯

**Comment by Loquacity** [comment]
> 💯

@clarkbw When do you expect the forking feature to go live?

**Comment by clarkbw** [comment]
Likely in the next 2 weeks

**Comment by Loquacity** [comment]
> Likely in the next 2 weeks

OK, happy to leave this here to gather more reviews for a while then. Please let the team know :)

**Comment by Loquacity** [comment]
> > Likely in the next 2 weeks
> 
> OK, happy to leave this here to gather more reviews for a while then. Please let the team know :)

@clarkbw Any updates here?

**Comment by clarkbw** [comment]
> @clarkbw Any updates here?

We're rolling out on Monday! In 11 or so hours.

**Comment by Loquacity** [comment]
> > @clarkbw Any updates here?
> 
> We're rolling out on Monday! In 11 or so hours.

Ready to publish now then?

File: cloud/operations.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+selecting the `Operations` tab.
+
+## General
+In the `General` section of the Operations dashboard, you can fork your service,
+reset the password, pause, or terminate the service.
+
+### Fork a service
+When you a fork a service, you create an exact copy of the service, including
+the underlying database. This allows you to create a copy that you can use for
+testing purposes, or to prepare for a major version upgrade.
```
Comment (alexeyklyukin) [inline cloud/operations.md:14]: Do we want to emphasize that the forked service has its own `tsdbadmin` user password, different from the origin?

**Review by alexeyklyukin — APPROVED** [review]
Looks good to me! (but see the comment).

File: cloud/operations.md (hunk 1 above)
Comment (clarkbw) [reply cloud/operations.md:14]: That's a good point. We show users the "next steps" / "cheatsheet" page but if they skip it saving the password it isn't the same as the parent database (by design).

**Pull request merged**